tests/src/test/scala/tests/SerializationSuite.scala

```diff
@@ -44,7 +44,7 @@ class SerializationSuite extends AnyFunSuite {
     assert(bsp4jValue.getTaskId.getParents.asScala == bsp4sValue.taskId.parents.get)
     assert(bsp4jValue.getEventTime == bsp4sValue.eventTime.get)
     assert(bsp4jValue.getDataKind == bsp4sValue.dataKind.get)
-    assert(bsp4jValue.getStatus.getValue == bsp4sValue.status.code)
+    assert(bsp4jValue.getStatus.getValue == bsp4sValue.status.id)
```

Comment:
Where does the change from `code` to `id` come from?

Reply:
It can be called value or code if you prefer

Reply:
But it will have to be the same for all enums